vipps-qr-api.md

```diff
@@ -209,6 +213,31 @@ The response will be similar to this, where the URL in the responseBody will be
 
 **Please note:** The `expiresIn` value is in seconds.
 
+### Polling 
+On OTP qrs, merchants will need to [poll](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#polling-guidelines) to get the result of the payment. In physical context, we recommend a polling interval of 1 second. Once a customer has scanned the QR, it is possible for merchants to show the customer a "waiting/spinner"-screen while they finish the payment. This will feel comforting for the user as they will get feedback that the payment is underway.
+
+Redirect from the QR-image to a waiting page has to be done by the merchant and does not happen automatically. Merchants will be notified of the QR scan by polling the `GET:/ecomm/v2/payments/{orderId}/details:` endpoint. Once the QR has been opened in the app, the `transactionId` field in `transactionLogHistory` will be set (it will not exist before the QR has been scanned). Once this field is set, you can safely show a "waiting for user" spinning screen on your POS while the user finish the payment.
```

Comment:
```suggestion
Redirect from the QR-image to a "waiting/spinner"-page has to be done by the merchant and does not happen automatically. Merchants must check the status of the QR code scan by polling the `GET:/ecomm/v2/payments/{orderId}/details:` endpoint. Once the QR code has been opened in the app, the `transactionId` field in `transactionLogHistory` will be set (it will not exist before the QR code has been scanned). Once this field is set, you can safely show a "waiting for user" spinning screen on your POS while the user finish the payment.
```